plugins/auth-backend-module-openshift-provider/package.json

```
@@ -0,0 +1,55 @@
+{
+  "name": "@backstage/plugin-auth-backend-module-openshift-provider",
+  "version": "0.1.0",
```

```suggestion
  "version": "0.0.0",
```

New packages start at `0.0.0`, the changeset will bump it to `0.1.0` and release it.